Disable Kerberos support, we don't have requests_gssapi or requests_kerberos

Index: Nagstamon/Servers/Generic.py
--- Nagstamon/Servers/Generic.py.orig
+++ Nagstamon/Servers/Generic.py
@@ -54,13 +54,13 @@ from Nagstamon.Config import (AppInfo,
 
 
 # requests_gssapi is newer but not available everywhere
-try:
-    # extra imports needed to get it compiled on macOS
-    import numbers
-    import gssapi.raw.cython_converters
-    from requests_gssapi import HTTPSPNEGOAuth as HTTPSKerberos
-except ImportError:
-    from requests_kerberos import HTTPKerberosAuth as HTTPSKerberos
+#try:
+#    # extra imports needed to get it compiled on macOS
+#    import numbers
+#    import gssapi.raw.cython_converters
+#    from requests_gssapi import HTTPSPNEGOAuth as HTTPSKerberos
+#except ImportError:
+#    from requests_kerberos import HTTPKerberosAuth as HTTPSKerberos
 
 try:
     from requests_ecp import HTTPECPAuth
@@ -272,8 +272,8 @@ class GenericServer(object):
                 self.session.auth = requests.auth.HTTPDigestAuth(self.username, self.password)
             elif self.authentication == 'ecp':
                 self.session.auth = HTTPECPAuth(self.idp_ecp_endpoint, username=self.username, password=self.password)
-            elif self.authentication == 'kerberos':
-                self.session.auth = HTTPSKerberos()
+#            elif self.authentication == 'kerberos':
+#                self.session.auth = HTTPSKerberos()
 
             # default to check TLS validity
             if self.ignore_cert:
